src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AutomaticEndAppealForRemissionRejectedTrigger.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.postsubmit;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.REMISSION_DECISION;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.DateProvider;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.RemissionDecision;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PostSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PostSubmitCallbackHandler;
+import uk.gov.hmcts.reform.iacaseapi.domain.service.Scheduler;
+import uk.gov.hmcts.reform.iacaseapi.infrastructure.clients.model.TimedEvent;
+
+@Component
+public class AutomaticEndAppealForRemissionRejectedTrigger implements PostSubmitCallbackHandler<AsylumCase> {
+
+    private final DateProvider dateProvider;
+    private final Scheduler scheduler;
+
+    @Value("${paymentAfterRemissionRejection.dueInMinutes}")
+    int schedule14DaysInMinutes;
```

Comment:
private?

Reply:
Scope needed is to the present class only.

Reply:
For @Value, private member with a setter injection (@Setter Lombok) is a better practice, keeping Class' strongly encapsulated